builds/misc/packages.yaml

```diff
@@ -222,33 +232,25 @@ jobs:
         inputs:
           workingDirectory: 'edgelet/hsm-sys/azure-iot-hsm-c/build'
           cmakeArgs: '--build . --config Release'
+      - powershell: edgelet/build/windows/package.ps1
+        displayName: Package iotedged, iotedge, libiothsm and docker/dockerd
       - task: CopyFiles@2
-        displayName: 'Copy iotedged to Artifact Staging'
+        displayName: 'Copy package to Artifact Staging'
         inputs:
-          SourceFolder: edgelet/target/release
+          SourceFolder: .
           Contents: |
-            *.exe
-            *.pdb
-            *.dll
+            *.cab
           TargetFolder: '$(build.artifactstagingdirectory)'
-      - task: CopyFiles@2
-        displayName: 'Copy libiothsm to Artifact Staging'
+      - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+        displayName: 'ESRP CodeSigning'
         inputs:
-          SourceFolder: 'edgelet/hsm-sys/azure-iot-hsm-c/build/Release'
-          Contents: |
-            *.dll
-            *.pdb
-          TargetFolder: '$(build.artifactstagingdirectory)'
-      - task: CopyFiles@2
-        displayName: 'Copy legal docs to Artifact Staging'
-        inputs:
-          SourceFolder: edgelet/contrib/docs
-          TargetFolder: '$(build.artifactstagingdirectory)/LICENSE'
-      - task: CopyFiles@2
-        displayName: 'Copy config to Artifact Staging'
-        inputs:
-          SourceFolder: edgelet/contrib/config/windows
-          TargetFolder: '$(build.artifactstagingdirectory)'
+          ConnectedServiceName: 'Azure IoT Edge Code Sign'
```

Comment:
The cert for this service connection is about to expire. Can you make this a variable and I'll get it updated to the correct value?

Reply:
Done.